drone/build/build_approve.go

```diff
@@ -2,10 +2,10 @@ package build
 
 import (
 	"fmt"
-	"strconv"
 
 	"github.com/drone/drone-cli/drone/internal"
 	"github.com/urfave/cli"
+	"strconv"
```

Comment:
strconv should be grouped with fmt per the Go [coding guidelines](https://github.com/golang/go/wiki/CodeReviewComments#imports)

> Imports are organized in groups, with blank lines between them. The standard library packages are always in the first group.